This is optional test dependency and we do not have it packaged yet.

--- anyio-4.12.1/pyproject.toml.orig
+++ anyio-4.12.1/pyproject.toml
@@ -59,10 +59,6 @@
     "pytest-mock >= 3.14",
     "trustme >= 1.0.0",
     "truststore >= 0.9.1; python_version >= '3.10'",
-    """\
-    uvloop >= 0.22.1; platform_python_implementation == 'CPython' \
-    and platform_system != 'Windows' \
-    """,
     """
     winloop >= 0.2.3; platform_python_implementation == 'CPython' \
     and platform_system == 'Windows' \
